test/mocks/mappings/claimedIdentity.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
could we remove this file for now please, to be added with the right endpoint instead of /claimedidentity once working on submit of check answers screen